playbooks/troubleshooting/troubleshooting_guide.adoc

```diff
@@ -0,0 +1,724 @@
+---
+---
+=  Troubleshooting Guide for OpenShift Enterprise 3.1
+
+Steve Ovens
```

Comment:
The authors and versions should now show up on the page. This is probably a formatting issue. See https://raw.githubusercontent.com/rhtconsulting/openshift-playbooks/master/playbooks/installation/installation.adoc as an example.